src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, Color color) {
+        setFigure(FigureType.RIGHT_RECTANGLE);
+        setFirstLeg(firstLeg);
+        setSecondLeg(secondLeg);
+        setColor(color);
+        double area = getCalculatedArea();
+        setArea(area);
+    }
+
+    public RightTriangle() {
+        setFigure(FigureType.RIGHT_RECTANGLE);
```

Comment:
The figure type is incorrectly set to `FigureType.RIGHT_RECTANGLE`. It should be `FigureType.RIGHT_TRIANGLE` to accurately reflect the type of figure being represented.